test/parallel/test-assert.js

```diff
@@ -587,8 +587,9 @@ function testBlockTypeError(method, block) {
     method(block);
     threw = false;
   } catch (e) {
-    assert.strictEqual(e.toString(),
-                       'TypeError: "block" argument must be a function');
+    assert.strictEqual(
+      e.toString(),
+      'TypeError[ERR_ARG_FUNCTION]: "block" argument must be a function');
```

Comment:
Can you use [common.expectsError](https://github.com/nodejs/node/tree/master/test#expectserrorcode-type-message) for this? Also watch out for https://github.com/nodejs/node/pull/11512